source/docs/articles/sites/code/applying-upstream-updates.md

```diff
@@ -22,8 +22,27 @@ Only use the one-click updates on the Dashboard to update your site's core. Do n
 8. Click **Visit Live Environment** to verify the update is live.
 ![The Code tab in the Pantheon site's dashboard's Live Environment, showing the upstream commits in the log as deployed.](/source/docs/assets/images/desk_images/357435.png)
 
+## Apply Upstream via Terminus
+
+If you prefer using the command line, you can apply updates with [Terminus](/docs/articles/local/cli/).
+
+### Update a Specific Site
+
+```
+terminus site upstream-updates apply --site=site --env=env --accept-upstream --updatedb

```

Comment:
I like pointing them to the command for help. Maybe leave out the `--accept-upstream` option and someone can find that option in the help?